src/expand.ts

```diff
@@ -249,3 +249,12 @@ async function expandStringHelper(input: string, opts: ExpansionOptions) {
     });
     return { result: finalString, didReplacement, circularReference };
 }
+
+export async function expandArrayOfStrings(inputs: string[], opts: ExpansionOptions): Promise<string[]> {
```

Comment:
A common naming simplification for arrays is to pluralize the variable/type that the array holds.
i.e. this function name could be simplified to `expandStrings` (plural).